tests/build-cluster-dns.yaml

```diff
@@ -0,0 +1,36 @@
+test_duration: 5760
+n_db_nodes: 3
+n_loaders: 1 
+n_monitor_nodes: 1
+user_prefix: 'build-cluster-dns'
```

Comment:
Please change the prefix to: cases-dns-VERSION.
The jenkins job should replace (sed) the VERSION with the correct version.